app/src/hooks/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './connectedWeb3'
+export * from '../contexts/connectedWeb3'
```

Comment:
Since we moved connectedWeb3 into the context it would be better if we exported it from there as well..